code/datums/religions.dm

```diff
@@ -277,7 +277,7 @@
 	deity_name = "Satan"
 	bible_name = "The Satanic Bible" //What I found on Google, ergo the truth
 	male_adept = "Magister"
-	female_adept = "Magistera"
+	female_adept = "Magistra"
```

Comment:
???

Reply:
oh uh shit

Reply:
im bad at github sorry that was supposed to be a different one

Reply:
a different pr

Reply:
oh god oh fuck

Reply:
just change it back on the same branch and stage and commit and push